app/models/__init__.py

```diff
@@ -827,20 +827,29 @@
             BUILD_ENVIRONMENT_KEY,
             CREATED_KEY,
             DEFCONFIG_FULL_KEY,
+            DEVICE_TYPE_KEY,
```

Comment:
Shouldn't some of these be mandatory keys?   The `BUILD_ID_KEY` was mandatory as it's required to find a matching build for the test results, so maybe the keys used to look up a build now should be mandatory instead.  I think these required keys are as per the `spec` used here:
https://github.com/kernelci/kernelci-backend/blob/master/app/utils/kci_test/__init__.py#L399

Reply:
Good point. Added mandatory fields